src/CalculatorUITestFramework/HistoryPanel.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+using OpenQA.Selenium.Appium;
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CalculatorUITestFramework
+{
+    public class HistoryPanel : ApplicationBase
+    {
+        public WindowsElement HistoryLabel => TryFindElementByAccessibilityId("HistoryLabel");
+        public WindowsElement HistoryListView => TryFindElementByAccessibilityId("HistoryListView");
+        public WindowsElement ClearHistoryButton => TryFindElementByAccessibilityId("ClearHistory");
+
+        /// <summary>
+        /// Opens the History Pane by clicking the History pivot label.
+        /// </summary>
+        public void OpenHistoryPanel()
+        {
+            HistoryLabel.Click();
+        }
+
+        /// <summary>
+        /// Gets all of the history items listed in the History Pane.
+        /// </summary>
+        /// <returns>A readonly collection of history items.</returns>
+        public ReadOnlyCollection<AppiumWebElement> GetAllHistoryListViewItems()
```

Comment:
Are the returned elements actually of type WindowsElement?

Reply:
No, the FindElementsByClassName returns a collection of AppiumWebElements. We can convert them if we want consistency.